src/functionalTest/resources/application-functional.yaml

```diff
@@ -35,4 +35,7 @@ resendInterval: 60
 resendInviteEnabled : true
 
 # delete organisation enabled
-deleteOrganisationEnabled : ${DELETE_ORG:true}
\ No newline at end of file
+deleteOrganisationEnabled : false
```

Comment:
why delete organisation disabling in the demo?

Reply:
Changed by mistake, it has been corrected here: https://github.com/hmcts/rd-professional-api/pull/725/commits/37af4c1f1e0374a3746f2c4c71374d27734d7acb